src/main/java/org/deepsymmetry/Wayang.java

```diff
@@ -265,4 +276,51 @@ public static void sendFrame() {
             }
         }
     }
+
+    /**
+     * Send a frame of pixels async, corresponding to whatever has been drawn in the image returned by open(),
+     * to the display.
+     *
+     * @throws LibUsbException       if there is a problem communicating.
+     * @throws IllegalStateException if the Push 2 has not been opened.
+     */
+    public static void sendFrameAsync() {
```

Comment:
I think this should take a callback argument, with an interface we define, that will get called when LibUsb calls our own callback, and pass along the success/failure of the frame send attempt.